crypto/sha/asm/keccak1600-x86_64.pl

```diff
@@ -345,6 +345,7 @@
 .size	__KeccakF1600,.-__KeccakF1600
 
 .globl	KeccakF1600
+.hidden KeccakF1600
```

Comment:
No, remove .globl instead. It's development artefact. I mean initially there was no SHA3_absorb or SHA3_squeeze in assembly, they were in C, so one needed entry point. But not anymore. BTW, if .globl had to be retained, preferred solution wouldn't be adding .hidden, but to arrange alternative local label and use that for internal calls. Less dependencies on execution format is better. In other words directives are *last* resort, in sense that if one can solve the problem without directives, it would be the preferred way.

Reply:
I tried the local label first and then we figured out the .hidden.  I've updated to remove the .globl instead.